### DISCUSSION TITLE
INSPIRE Geoportal GeoNetwork harvesting console interruption

### DISCUSSION THREAD

**Discussion by jescriu** [INSPIRE-MIF/helpdesk-geoportal#140] [Announcements]
Dear representatives / managers of national geoportal catalogues,

On Friday 17 March 2023 the administration users to manage your harvesters in the new [INSPIRE Geoportal harvesting console based on GeoNetwork](https://jrc-geoportal-internal.geocat.live/geonetwork) will be temporarily disabled due to maintenance works in the backend.

The service will be in principle re-established again by 21 March.

Apologies for any inconveniences this interruption may cause.

JRC INSPIRE Geoportal Team

**Comment by jescriu** [comment]
As previously announced on 13 March, the administration users to manage the harvesters included in the [INSPIRE Geoportal harvesting console based on GeoNetwork](https://jrc-geoportal-internal.geocat.live/geonetwork) have been disabled today.

The service will be in principle re-established again by 21 March.

JRC INSPIRE Geoportal Team

**Reply by bcosyn** [reply]
Seems that https://jrc-geoportal-internal.geocat.live/geonetwork is working again . But is admin access still disabled? If so when will it be enabled again.

**Comment by jescriu** [comment]
Dear @bcosyn,
I regret to inform you that the maintenance tasks will remain at least for an additional week.
The INSPIRE Geoportal team will provide news using this thread when the service is re-established.  
Excuses for any inconvenience this may cause.

**Comment by jescriu** [comment]
Dear representatives / managers of national geoportal catalogues,

The [INSPIRE Geoportal harvesting console based on GeoNetwork](https://jrc-geoportal-internal.geocat.live/geonetwork) is already available. The national administration users have been enabled again.

Thank you for your patience.

**Comment by PB-GNM** [comment]
I have tried 3 times this week to harvest the Dutch national catalogue (https://www.nationaalgeoregister.nl/geonetwork/srv/dut/csw-inspire), but after 142 seconds it reports an error:  https://jrc-geoportal-internal.geocat.live/geonetwork/srv/eng/admin.harvester.log?file=logs%2Fharvester_csw2_NL_202305041328.log.
What could be the problem?

Pieter Bresters,
Geonovum

**Reply by manilly** [reply]
Same problem for AT: https://jrc-geoportal-internal.geocat.live/geonetwork/srv/eng/admin.harvester.log?file=logs%2Fharvester_csw2_AT_202305041346.log

@jescriu @
